src/pages/index.js

```diff
@@ -41,6 +47,9 @@ export default () => {
   const classes = useStyles()
   const theme = useTheme()
   const matches = useMediaQuery('(min-width:640px)')
+  const active = useMediaQuery('(min-width:960px)')
+  const FontLocation = active ? { fontSize: 30 } : null
+  const FontCalendar = active ? { fontSize: 30 } : null
```

Comment:
Why you need this @Rajani169 when you defined the styling above in the classes, implement the styling for both venue and date using classes only, implement the media queries in classes.

Reply:
Done!!